pandas/core/indexes/range.py

```diff
@@ -673,6 +681,12 @@ def __floordiv__(self, other):
                     start, start + 1, 1, name=self.name)
         return self._int64index // other
 
+    def all(self):
+        return 0 not in self._range
+
+    def any(self):
```

Comment:
can you type

Reply:
I will need to wait for this until #26581 is merged, as mypy doesn't understand/see attributes defined in ``__new__`` or ``_simple_new`` . If that one is merged, I can go through this again wrt. typing.